src/content/en/2021/seo.md

```diff
@@ -1,20 +1,723 @@
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=819909313&format=interactive",
+   sheets_gid="412947118",
+   sql_file="image-alt-stats.sql"
+   )
+}}
+
+The image loading property affects how user agents prioritize rendering and display of images on the page. It may impact user experience and page load performance, both of which impact SEO success.
+
+- 5% of pages don't use any image loading property.
+- 6% of pages use loading="lazy" which delays loading an image until it is close to being in the viewport.
+- 8% of pages use loading="eager" which loads the image as soon as the browser loads the code.
+- 1% of pages use invalid loading properties.
+- 1% of pages use loading="auto" which uses the default browser loading method.
+
+{{ figure_markup(
+   image="image-loading-property-usage.png",
+   caption="Image loading property usage.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1305654777&format=interactive",
+   sheets_gid="55531578",
+   sql_file="image-loading-property-usage.sql"
+   )
+}}
+
+### Word Count
+
+The number of words on a page isn't a ranking factor, but the _way_ pages deliver words can profoundly impact rankings. Words can be in the raw page code or the rendered content.
+
+First, rendered page content: "Rendered" is the content of the page after the browser has executed all Javascript and any other code that modified the DOM or CSSOM.
+
+- The median _rendered_ desktop page contains 425 words, versus 402 words in 2020.
+- The median _rendered_ mobile page contains 367 words, versus 348 words in 2020.
+- Rendered mobile pages contain 13.6% fewer words than rendered desktop pages. Note that Google is a mobile-only index. Content not on the mobile version may not get indexed.
+
+{{ figure_markup(
+   image="visible-rendered-words-percentile.png",
+   caption="Visible words rendered by percentile.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=833732027&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats-by-percentile.sql"
+   )
+}}
+
+Second, raw page content: "Raw" is the content of the page before the browser has executed Javascript or any other code that modified the DOM or CSSOM. It's the "raw" content delivered and visible in the source code.
+
+- The median _raw_ desktop page contains 369 words, versus 360 words in 2020.
+- The median _raw_ mobile page contains 321 words, versus 312 words in 2020.
+- Raw mobile pages contain 13.1% fewer words than rendered desktop pages. Note that Google is a mobile-only index. Content not on the mobile desktop may not get indexed.
```

Comment:
`Content not on the mobile desktop may not get indexed.` is a bit confusing to me.
```suggestion
- Raw mobile pages contain 13.1% fewer words than rendered desktop pages. Note that Google is a mobile-only index. Content not on the mobile HTML version may not get indexed.
```

Reply:
Weird, that's not what it says in the Google doc. Maybe someone had suggested a change and that carried over into the markdown.